### PR TITLE
MC-131: When a category is moved in Akeneo, change is now correctly pass...

### PR DESCRIPTION
**Pull request by damien-carcel**
Bug fix: [yes]
Feature addition: [no]
Backwards compatibility break: [no]
Phpspec specification passes: [no]
Checkstyle issues: [no]
ChangeLog updated: [yes]
Fixes the following issue:

### CODE REVIEW

File: Normalizer/CategoryNormalizer.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class CategoryNormalizer extends AbstractNormalizer
      * @param CategoryMappingManager $categoryMappingManager
      */
     public function __construct(
-        ChannelManager $channelManager,
+        ChannelManager         $channelManager,
```
Comment (BitOne) [inline Normalizer/CategoryNormalizer.php]: Argh, you have to refrain yourself to do that on a bug fix only PR.
If you want to do that, do that on another PR, please.

File: Normalizer/CategoryNormalizer.php [hunk 2]
```diff
@@ -180,6 +180,7 @@ protected function getNormalizedNewCategory(CategoryInterface $category, array $
 
     /**
      * Get update normalized categories
+     *
```
Comment (BitOne) [inline Normalizer/CategoryNormalizer.php]: Same...

Comment (damien-carcel) [inline Normalizer/CategoryNormalizer.php]: Sorry, I will not do it again, promise.
